test/assets/text-alignment.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Lato">
+  <style>
+  body {
+    font-family: 'Lato', serif;
+    font-size: 15px;
+  }
+  div.a {
+    line-height: 0.5;
+  }
+  </style>
+</head>
+<body>
+  <div class="a">
+    <div>aaaaa..............................................................................|111</div><br>
```

Comment:
We don't have consistent font rendering across operating systems, so this test is going to be tricky to make work. I am not an expert on fonts, but I imagine there is more than just the hinting that will make this test not pixel perfect.

Reply:
Yeah, that's consistent with my experiences thus far. How would you recommend testing the change?

Reply:
(assuming this is even the approach we decide on)

Reply:
We could launch the page and headless and headful, and check that the screenshots are the same. Look at the headful.spec.js for examples of that.